src/main/java/frc/robot/RobotContainer.java

```diff
@@ -131,249 +131,249 @@ private void setUpSubsystems() {
-                m_vision = new Vision(visionIO);
-                m_Led = new LED(m_vision, m_indexer);
+                climber = new Climber(climberIO);
+                arm = new Arm(armIO);
+                shooter = new ShooterSubsystem(shooterIO);
+                indexer = new Indexer(indexerIO);
+                intake = new Intake(intakeIO);
+                vision = new Vision(visionIO);
+                Led = new LED(vision, indexer);
 
         }
 
         // sets up auton commands
         private void setUpAuton() {
                 NamedCommands.registerCommand("intake", intakeWithHeightRestriction());
                 NamedCommands.registerCommand("Intake", intakeWithHeightRestriction());
-                NamedCommands.registerCommand("intake for time", intakeForTime(m_intake, m_indexer));
-                NamedCommands.registerCommand("SHORT intake for time", shortIntakeForTime(m_intake, m_indexer));
+                NamedCommands.registerCommand("intake for time", intakeForTime(intake, indexer));
+                NamedCommands.registerCommand("SHORT intake for time", shortIntakeForTime(intake, indexer));
                 NamedCommands.registerCommand("AimToTarget", Commands.print("aimed to target!"));
-                NamedCommands.registerCommand("SetArmPosition", makeSetPositionCommandAuton(m_arm, 0.74));
-                NamedCommands.registerCommand("Set Arm Wingleft", makeSetPositionCommandAuton(m_arm, 0.785));
-                NamedCommands.registerCommand("SetArmDown", makeSetPositionCommandAuton(m_arm, 0.335));
+                NamedCommands.registerCommand("SetArmPosition", makeSetPositionCommandAuton(arm, 0.74));
+                NamedCommands.registerCommand("Set Arm Wingleft", makeSetPositionCommandAuton(arm, 0.785));
+                NamedCommands.registerCommand("SetArmDown", makeSetPositionCommandAuton(arm, 0.335));
                 NamedCommands.registerCommand("AutoShoot", outtakeAndShootAfterDelay());
                 NamedCommands.registerCommand("intake and outtake", intakeAndOuttake());
                 NamedCommands.registerCommand("outtake", outtake());
 
-                m_autoChooser = AutoBuilder.buildAutoChooser();
-                SmartDashboard.putData("Autos/Selector", m_autoChooser);
+                autoChooser = AutoBuilder.buildAutoChooser();
+                SmartDashboard.putData("Autos/Selector", autoChooser);
 
         }
 
         // Configure default commands
         private void configureDefaultCommands() {
                 // default command for the shooter: do nothing
-                m_shooter.setDefaultCommand(
+                shooter.setDefaultCommand(
                                 new RunCommand(
-                                                () -> m_shooter.setMotor(0),
-                                                m_shooter).withName("drive default"));
+                                                () -> shooter.setMotor(0),
```

Comment:
this isn't up to standards for a code rewrite! It should be using the command generated from the subsystem's command factory, not a run command